specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/examples/BudgetsList.json

```diff
@@ -0,0 +1,159 @@
+                ],
+                "meters": [
+                  "fe167397-a38d-43c3-9bb3-8e2907e56a41"
+                ],
+                "tags": {
+                    "category": [
+                        "Dev",
+                        "Prod"
+                    ],
+                    "department": [
+                        "engineering",
+                        "sales"
+                    ]
+                }
+              },
+              "currentSpend": {
+                "amount": 80.89,
+                "unit": "USD"
+              },
+              "notifications": {
+                "Actual_GreaterThan_80_Percent": {
+                  "enabled": true,
+                  "operator": "GreaterThan",
+                  "threshold": 80,
+                  "contactEmails": [
+                    "johndoe@contoso.com",
+                    "janesmith@contoso.com"
+                  ],
+                  "contactRoles": [
+                    "Contributor",
+                    "Reader"
+                  ]
+                },
+                "Actual_GreaterThanOrEqualTo_90_Percent": {
+                  "enabled": true,
+                  "operator": "GreaterThanOrEqualTo",
+                  "threshold": 90,
+                  "contactEmails": [
+                    "johndoe@contoso.com",
+                    "janesmith@contoso.com"
+                  ],
+                  "contactRoles": [
+                    "Contributor",
+                    "Reader"
+                  ],
+                  "contactGroups": [
+                    "/subscriptions/{subscription-id}/resourceGroups/MPDEVTESTRG/providers/microsoft.insights/actionGroups/SampleActionGroup"
+                  ]
+                }
+              }
+            }
+          },
+          {
+            "id": "subscriptions/{subscription-id}/providers/Microsoft.Consumption/budgets/TestBudget",
+            "name": "TestBudget",
+            "type": "Microsoft.Consumption/budgets",
+            "eTag": "\"1d34d012214157f\"",
+            "properties": {
+              "category": "Cost",
+              "amount": 600.65,
+              "timeGrain": "Monthly",
+              "timePeriod": {
+                "startDate": "2017-10-01T00:00:00Z",
+                "endDate": "2018-10-31T00:00:00Z"
+              },
+              "filters": {
+                "resourceGroups": [
+                  "MPDEVTESTRG"
+                ],
+                "resources": [
+                  "/subscriptions/{subscription-id}/resourceGroups/MPDEVTESTRG/providers/Microsoft.Compute/virtualMachines/MSAWSIFT2",
+                  "/subscriptions/{subscription-id}/resourceGroups/MPDEVTESTRG/providers/Microsoft.Compute/virtualMachines/platformcloudplatformGeneric1"
+                ],
+                "meters": [
+                  "fe167397-a38d-43c3-9bb3-8e2907e56a41"
```

Comment:
Looks like evidence of the comment above, that we could hint at the format of `meterId`s.

Also, for what it's worth, I recommend stripping or obfuscating GUIDs so that nothing specific to your service is available in the configs.

Reply:
Very good point. I have removed the actual GUIDs from all examples and added default GUID.